.github/workflows/main.yml

```diff
@@ -12,14 +12,10 @@ jobs:
         steps:
             - uses: actions/checkout@v2
             - uses: actions/setup-node@v1
-            - name: Install yarn & Dependencies
-              run: |
-                  npm install -g yarn
-                  yarn install
+            - name: Install Dependencies
+              run: yarn install
             - name: Check Formatting
-              run: |
-                  yarn lint
-                  yarn format:check
+              run: yarn lint && yarn format:check
```

Comment:
I don't know whether these `steps` abort on first failure, but if they don't, it wouldn't hurt for these to be two separate steps (otherwise a lint failure will hide formatting errors).

Reply:
Oh good call, I didn't think about that